dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -570,6 +625,7 @@ def test_correct_total_sample_size_and_counts_and_mutability(self):
         file_path = os.path.join(test_root_path, 'data', 'csv/empty_rows.txt')
         data = pd.read_csv(file_path)
         profiler_options = ProfilerOptions()
+        profiler_options.structured_options.multiprocess.is_enabled = False
```

Comment:
so, we never test if the multiprocessing running for any test?

Reply:
^ I'm personally of the opinion multiprocessing should run on all tests, unless otherwise specified as it's the default behavior.

Reply:
it requires significantly more time to run the tests. If we want to revery we can.